### ISSUE TITLE
[feat]: File Upload – Add ability to provide `aria-label` to the input

### ISSUE THREAD

**Issue by besuhoff** [sadmann7/diceui#263]
### Feature description

Currently it's not possible to pass `aria-label` to the file input in File Upload. It has an `aria-labelledby` though, but it points to a label with `sr-only` class, which makes it invisible for non screen readers: [docs/registry/bases/radix/ui/file-upload.tsx#L653](https://github.com/sadmann7/diceui/blob/main/docs/registry/bases/radix/ui/file-upload.tsx#L653). `axe-core` validation rule `label-title-only` fails on the input field due to that (see [my report in their repository](https://github.com/dequelabs/axe-core/issues/5014) for the context)

It would be good if the `aria-label` parameter passed to root component would be assigned to the input, not the wrapping element. That could improve the accessibility of the whole thing.

### Before submitting

- [x] I've made research efforts and searched the documentation
- [x] I've searched for existing issues and PRs
